### PR TITLE
Engdocs 633

### PR DESCRIPTION
**Pull request by aevesdocker**
New user docs for Extensions.

### CODE REVIEW

File: desktop/extensions.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+description: Extensions
+keywords: Extensions, Docker Desktop, Linux, Mac, Windows
+title: Extensions
```
Comment (usha-mandya) [inline desktop/extensions.md:4]: Suggest: Docker Extensions

File: desktop/extensions.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+toc_min: 1
+toc_max: 2
+---
+
+Docker Extensions lets you use third-party tools within Docker Desktop to extend its functionality. There is no limit to the number of extensions you can install.
+
+Docker Community members and partners can use our [SDK](https://docker-desktop-extensions.netlify.app/){:target="_blank" rel="noopener" class="_"}  to create new extensions.
```
Comment (usha-mandya) [inline desktop/extensions.md:11]: The link currently points to the Netlify site. @gtardif Wondering if we can also have the SDK docs ready when the user manual goes live?

File: desktop/extensions.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+toc_max: 2
+---
+
+Docker Extensions lets you use third-party tools within Docker Desktop to extend its functionality. There is no limit to the number of extensions you can install.
+
+Docker Community members and partners can use our [SDK](https://docker-desktop-extensions.netlify.app/){:target="_blank" rel="noopener" class="_"}  to create new extensions. 
+
+> **Preview**
+>
+>The Docker Extensions feature is currently offered as a Preview. We recommend you do not use this in production environments.
```
Comment (usha-mandya) [inline desktop/extensions.md:15]: nit: Add space after `>`

File: desktop/extensions.md (hunk 2 above)
Comment (aevesdocker) [reply desktop/extensions.md:11]: Currently reviewing the SDK docs now. They won't be ready to be released today along with the user manual. Was going to create a separate ticket to update with the new link when the SDK docs are ready.

File: desktop/extensions.md (hunk 3 above)
Comment (usha-mandya) [inline desktop/extensions.md:15]:
```suggestion
> The Docker Extensions feature is currently offered as a Preview. We recommend that you do not use Extensions in production environments.
```

File: desktop/extensions.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+3. Click **Install**.<br>
+From here, you can click **Open** to access the extension or install additional extensions. The extension also appears in the menu bar.
+
+## Enable or disable extensions available in the Marketplace
+
+ Docker Extensions are switched on by default. To change your settings:
+
+1. Navigate to  **Settings**, or **Preferences** if you're a Mac user.
+2. Select the **Extensions** tab.
+3. Next to **Enable Docker Extensions**, select or deselect the checkbox to to set your desired state.
```
Comment (usha-mandya) [inline desktop/extensions.md:48]:
```suggestion
3. Next to **Enable Docker Extensions**, select or clear the checkbox to set your desired state.
```

File: desktop/extensions.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+4. In the bottom-right corner, click **Apply & Restart**.
+
+## Enable or disable extensions not available in the Marketplace
+
+You can install Docker Extensions through the Marketplace or through the Extensions SDK tools. You can choose to only allow published extensions (that have been published in the Extensions Marketplace).
+
+1. Navigate to **Settings**, or **Preferences** if you're a Mac user.
+2. Select the Extensions tab.
```
Comment (usha-mandya) [inline desktop/extensions.md:56]:
```suggestion
2. Select the **Extensions** tab.
```

File: desktop/extensions.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+3. Next to **Allow only extensions distributed through the Docker Marketplace**, select or deselect the checkbox to set your desired state.
```
Comment (usha-mandya) [inline desktop/extensions.md:57]:
```suggestion
3. Next to **Allow only extensions distributed through the Docker Marketplace**, select or clear the checkbox to set your desired state.
```

File: desktop/extensions.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+4. In the bottom-right corner, click **Apply & Restart**.
+
+## Update an extension
+You can update Docker Extensions outside of Docker Desktop releases. To update an extension to the latest version:
+
+1. From the menu bar, select the ellipsis to the right of **Extensions**.
```
Comment (usha-mandya) [inline desktop/extensions.md:63]: Suggest we add the navigation to go to the Extensions tab in Docker Dashboard as a first step

File: desktop/extensions.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+2. Click **Manage Extensions**.
+If an extension has a new version available, an **Update** button is visible.
```
Comment (usha-mandya) [inline desktop/extensions.md:65]:
```suggestion
If an extension has a new version available, it displays an **Update** button.
```

File: desktop/extensions.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+3. Click **Update**.
+
+## Uninstall an extension
+ You can uninstall an extension at any time. 
+ 
+ > **Note**  
+ >
+ >Any data used by the extension that is stored in a volume must be manually deleted.
```
Comment (usha-mandya) [inline desktop/extensions.md:73]:
```suggestion
 > Any data used by the extension that is stored in a volume must be manually deleted. 
```

**Comment by usha-mandya** [comment]
@aevesdocker Thanks for the PR. Added some minor suggestions. Looks good overall.

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**